.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Seems like this has tabs rather than the 2-space setting in .editorconfig

Reply:
Fixed.